### DISCUSSION TITLE
How to translate the labels in the Bottom Tool Bar

### DISCUSSION THREAD

**Discussion by RajathShanmugaiah** [react-page/react-page#1063] [Q&A]
![translate](https://user-images.githubusercontent.com/63197365/131665933-1f8c2ae2-f6dd-4504-9fff-2c692fd3ff33.png)
I want to basically translate the labels present in the bottom tool bar. Do you have any suggestion to handle it?

**Comment by rexebin** [comment]
react-page Editor component takes a prop called `uiTranslator`: https://react-page.github.io/docs/#/editor?id=uitranslator 

you may try and see if this is what you needed.

**Reply by RajathShanmugaiah** [reply]
@rexebin I followed the uiTranslator Docs earlier 

```
const uiTranslator = useCallback((label?: string| any) => {
      if (Translations[label] !== undefined) {
        return Translations[label];
      }
      return `${label}(to translate)`;
    }, [Translations]);
```
//editor 
```<Editor
          cellPlugins={cellPlugins}
          value={value}
          onChange={setValue}
          components={components}
          languages={LANGUAGES}
          lang={language}
          uiTranslator={uiTranslator}
        />```
Here is my sample code 
Value is containing the data set

So basically using the uiTranslator function its only able to translate the buttons of the toolbar and not the input field labels in the bottom toolbar

**Comment by rexebin** [comment]
In this case, raising an issue on the repo and see if there is anything can
be done.
